site/app/templates/submission/homework/SubmitBox.twig

```diff
@@ -21,6 +21,10 @@
         </div>
     </header>
 
+    {% if part_names|length > 1 and one_part_only %}
+        <h4 style="color: red;">Only first non-empty part will be graded!</h4>
```

Comment:
The term "part" may not be clear to the student user.

Let's rephrase:
"Please drag and drop your files into only one area below.  Files submitted to additional areas will not be graded."

Reply:
Done!